sign/sign.go

```diff
@@ -16,13 +16,19 @@ import (
 var (
 	// ErrorSignatureValidationFailed is returned when a signature is not valid.
 	ErrorSignatureValidationFailed = errors.New("signature validation failed")
+	ErrorCannotSignEmptyBody       = errors.New("cannot sign empty body")
+	ErrorInvalidPersonaTag         = errors.New("invalid persona tag")
+	ErrorInvalidNamespace          = errors.New("invalid namespace")
 )
 
+const SystemPersonaTag = "SystemPersonaTag"
```

Comment:
should comment some stuff about how this is a reserved tag for internal/sys txs

Reply:
done